tst/EnergyPlus/unit/Datasets.unit.cc

```diff
@@ -0,0 +1,159 @@
+// C++ Headers
+#include <fstream>
+#include <string>
+#include <vector>
+#include <sstream>
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// Fixtures, etc.
+#include "Fixtures/EnergyPlusFixture.hh"
+
+namespace EnergyPlus {
+
+	typedef EnergyPlusFixture DataSetFixture;
+
+	std::vector<std::string> getAllLinesInFile( std::string filePath ) {
```

Comment:
@Myoldmopar No, but it would be trivially easy to implement. And it would be better than having another function read the IDF (which process_idf already does). I didn't implement this to begin with because I didn't want it to get abused. We can talk about it this week.